packages/react/src/Input/Input.tsx

```diff
@@ -32,24 +37,26 @@ export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
         className,
         'relative flex items-center rounded-md border-[1px] border-b-[3px] focus-within:-ml-[2px] focus-within:-mt-[2px] focus-within:border-[3px] focus-within:shadow',
         {
-          'border-gray-dark focus-within:border-blue': !isInvalid,
+          'focus-within:border-blue-dark border-black': !isInvalid,
           'border-red focus-within:border-red': isInvalid,
           'w-fit': size != null,
           'w-full': size == null,
         },
       )}
     >
-      {prefix && <span className="text-gray pl-4">{prefix}</span>}
+      {prefix}
```

Comment:
Spørsmål: Vil vi wrappe prefix/suffix og legge til padding left/right eller legge til padding left/right i diven over utifra om prefix eller suffix er lagt til

```js
<div className={cx({ 'pl-4': prefix })}>
   {prefix}
    <input />
</div>
```

eller

```js
<div>
  {prefix && <div className='pl-4'>{prefix}</div>}
  <input />
</div>
```

Heller mer mot den første enn den andre. 

Eller skal vi selv la brukeren legge til padding?

Reply:
tenker at padding burde håndteres for deg. 99% av bruken kommer til å være å slenge inn et ikon.